src/components/Card/Card.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import axios from 'axios';
+import { Root, Result } from '../../types/TypeMovieDB';
+import styles from './card.module.scss';
+
+const Card: React.FC = () => {
+  const [movies, setMovies] = React.useState<Result[]>([]);
+
+  const IMAGE_PATH = 'https://image.tmdb.org/t/p/w500';
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const result = await axios(
+        `${process.env.REACT_APP_BASE_URL}/movie/popular?api_key=${process.env.REACT_APP_API_KEY}&language=tr&page=1`
+      );
+      setMovies(result.data.results);
+    };
+    fetchData();
+  }, []);
+
+  console.log(movies);
+
+  return (
+    <>
+      {movies.map((movie: Result) => {
```

Comment:
burada her bir movie için type belirtiyorsak, onun ismini daha spesifik verelim. Result birazcık API'dan dönen tüm response için gibi duruyor.